modules/ROOT/pages/anypoint-mq/2.x/anypoint-mq-listener.adoc

```diff
@@ -230,6 +230,26 @@ In many cases there's a single common service between the processing of messages
 
 For this scenario, both subscribers stop polling for messages as soon as the error count reaches the `errorsThreshold="10"` value, counting both `FTP:RETRY_EXHAUSTED` and `HTTP:SERVICE_UNAVAILABLE` errors. When the `tripTimeout` elapses, one of the subscribers polls for a message and uses it to test the circuit, enabling the polling for both subscribers if the processing of that message succeeds.
 
+
+== FIFO Queues
+
+By default, FIFO queues support a single consumer only. When one consumer is accessing a message, all other consumers are blocked until the first batch is processed. No messages are delivered until all in-flight messages are acknowledged or not acknowledged. With message groups, multiple consumers can access messages in a FIFO queue. In this case, a single consumer accesses messages in a group and message order is preserved within the message group.
+
+=== FIFO Queues and Clustering  
+
+FIFO queues behave the same in a clustered environment as in a non-clustered environment.
+
+FIFO queues consume messages in the specified order. After the message is consumed, any further message processing can be distributed to other nodes. In this case, message order might be lost during message processing, for example, if one server processes messages faster than another or if the consumer acknowledges a message before it is processed.
```

Comment:
> message order might be lost during message processing, for example, if one server processes messages faster than another or if the consumer acknowledges a message before it is processed.

I'd change to `message order might be lost during message processing, if the consumer acknowledges a message before it is fully processed`.

Processing speed matters on multi-thread approach, not between different workers.